mteb/models/model_implementations/tarka_models.py

```diff
@@ -56,3 +339,39 @@
     memory_usage_mb=576,
     citation=Tarka_Embedding_150M_V1_CITATION,
 )
+
+tark_embedding_350_v1_kwargs = dict(
+    device="cuda",  # use a gpu
+    model_kwargs={
+        "attn_implementation": "flash_attention_2",
+        "device_map": "cuda",
+        "torch_dtype": "bfloat16",
+    },  # use low-precision
```

Comment:
Can you remove `device` and `device_map`? I think users can select device by themselves. 
```suggestion
    model_kwargs={
        "attn_implementation": "flash_attention_2",
        "torch_dtype": "bfloat16",
    },  # use low-precision
```